src/intrprtr.c

```diff
@@ -4023,8 +4034,11 @@ void             IntrAssertAfterCondition ( void )
 void             IntrAssertEnd2Args ( void )
 {
       /* ignore or code                                                      */
-    SKIP_IF_RETURNING();
-    if ( STATE(IntrIgnoring)  > 2 ) { STATE(IntrIgnoring) -= 2; return; }
+      INTERPRETER_PROFILE_HOOK(2);
```

Comment:
Huh, the indentation looks off, at least compared to what was there before?!?

Reply:
the old formatting was 2 space (which nothing else in this file is), that's clang-format moving it to 4 spaces.